rules/gather_licenses_info.bzl

```diff
@@ -135,15 +135,25 @@ def write_licenses_info(ctx, deps, json_out):
             This requires that you have run the gather_licenses_info
             aspect over them
       json_out: output handle to write the JSON info
+
+    Returns:
+      A list of License File objects for each of the license paths referenced in the json.
     """
-    licenses = []
+    licenses_json = []
+    licenses_files = []
     for dep in deps:
         if TransitiveLicensesInfo in dep:
-            licenses.extend(licenses_info_to_json(dep[TransitiveLicensesInfo]))
+            transitive_licenses_info = dep[TransitiveLicensesInfo]
+            licenses_json.extend(licenses_info_to_json(transitive_licenses_info))
```

Comment:
This seems to be from a different PR.

Reply:
No, it is this PR. The Files returned from licenses_info_to_json are added to licenses_json which is returned by write_licenses_info.

Reply:
This is needed, so that downstream customers have an alternative to using TransitiveLicensesInfo directly, so it can be deprecated.

Reply:
Well... It's not going to matter much.   gather_licenses_info is going to be deprecated too over time.
Google will use it internally for a while, but I'll b doing all examples around user PacakgeInfo and user defined providers.